### ISSUE TITLE
CKV_AZURE_78: false positive for "siteConfig/ftpsState: 'Disabled'"

### ISSUE THREAD

**Issue by arnaud-tincelin** [bridgecrewio/checkov#6441]
**Describe the issue**

Language: Bicep
Check ID: CKV_AZURE_78

**Examples**
Extract of my code
```Bicep
resource api 'Microsoft.Web/sites@2022-09-01' = {
  name: name
  location: location
  kind: 'api'

  identity: {
    type: 'SystemAssigned'
  }

  properties: {
    serverFarmId: serverFarmId
    clientAffinityEnabled: false
    clientCertEnabled: false
    httpsOnly: true
    siteConfig: {
      ftpsState: 'Disabled'
     ...
    }
}
```

I would expect no issue on the FTP feature as it is set to `Disabled` but I am having

```
Tool
checkov

Rule ID 
CKV_AZURE_78

Description
Ensure FTP deployments are disabled
Resource: Microsoft.Web/sites.api
```

**Version (please complete the following information):**

We are using github action `bridgecrewio/checkov-action@v12`, so I expect it to use the latest

**Additional context**

Looking at https://github.com/bridgecrewio/checkov/blob/main/checkov/arm/checks/resource/AppServiceFTPSState.py,

I am guessing this line

```py
    def get_inspected_key(self) -> str:
        return "siteConfig/ftpsState"
```

should be

```py
    def get_inspected_key(self) -> str:
        return "properties/siteConfig/ftpsState"
```

**Comment by Saarett** [comment]
Hi @arnaud-tincelin , thank you for reaching out.
Seems like an easy fix, would you like to contribute a fix? 🙂

**Comment by ChanochShayner** [comment]
Closing, @arnaud-tincelin thank you for the contribution!
